playbooks/rhv/inventory.example

```diff
@@ -0,0 +1,61 @@
+[all:vars]
+openshift_rhv_dns_zone=  # example.com
+
+[OSEv3:children]
+nodes
+masters
+etcd
+lb
+
+[OSEv3:vars]
+# General variables
+ansible_ssh_user=root
```

Comment:
I saw you pushed some patches. This is still not updated.

Reply:
Thanks, the comment flow here makes for a horrible task list.

Reply:
Agree!